stdlib/_operator.pyi

```diff
@@ -105,22 +106,10 @@ class attrgetter(Generic[_T_co]):
 
 @final
 class itemgetter(Generic[_T_co]):
-    # mypy lacks support for PEP 646 https://github.com/python/mypy/issues/12280
-    # So we have to define all of these overloads to simulate unpacking the arguments
     @overload
-    def __new__(cls, item: _T_co) -> itemgetter[_T_co]: ...
+    def __new__(cls, item: _T_co) -> itemgetter[_T_co]: ...  # type: ignore[overload-overlap]
```

Comment:
This overload overlaps with the second with "0-length `*items`" but both mypy and pyright are able to pick this overload correctly in this case.

Reply:
A slightly more correct way would be to add a second required item to the second overload, since the tuple variant always requires two items. Also, while we're here:

```suggestion
    def __new__(cls, __item: _T_co) -> itemgetter[_T_co]: ...  # type: ignore[overload-overlap]
```

Reply:
Indeed item is positional-only in the C implementation but not in the Python implementation. I applied this patch but with the type ignore comment removed